content/applications/general/email_communication/email_common.rst

```diff
@@ -0,0 +1,123 @@
+
+Incoming emails
+===============
+
+When you have an issue with incoming emails, there might not be an indication per se in Odoo. This is the client who tries to contact a database who will get a bounce (most of the time 550: mailbox unavailable).
+
+Emails are not received
+-----------------------
+
+Depending on the platform you are using: 
+ - The **Odoo.sh** users can find their live logs on the folder ~/logs/.
+ - The folder ~/logs/ (preferably accessed by the command line) of an Odoo.sh contains a list of files containing the logs of the database. The log files are created everyday at 4:00 AM UTC. The two last days are not compressed, while the older ones are, in order to gain space. The naming of the files are decremental, which means that the file named .1 is the one from yesterday and the file .2 is the day before yesterday. See the following Odoo.sh documentation about `logs <https://www.odoo.com/documentation/master/administration/odoo_sh/getting_started/branches.html?#logs>`_. Use the command ``grep`` and ``zgrep`` (for the compressed ones) to search through the files. 
+ - The **SaaS** users won’t have access to their logs. However you can still contact `Odoo Support <https://www.odoo.com/help>`_ , if you have a recurring issue with the same client or domain.
+
+Get helped from support
+-----------------------
+
+In order to get helped efficiently, please provide as much information as possible. Here is a list of what can be helpful:
+ - The **EML** of the email, stating for *Electronic Mail*, is the file format containing all the technical information required for an investigation. The documentation of your own email provider might help you on how to get your EML files. Once you get the EML of the email, adding it in the attachment of your ticket is the most efficient way for us to investigate. The support will mainly focus on redundant issues.
+
+
+.. seealso:: Here are links to the documentation of some common providers:
+
+   - `GMail <https://support.google.com/mail/answer/29436>`_
+   - `Outlook <https://support.microsoft.com/en-us/office/view-internet-message-headers-in-outlook-cd039382-dc6e-4264-ac74-c048563d212c#tab=Web>`_
+
+- The exact flow you are following in order to normally receive those emails in Odoo. Here are examples of questions whose answers can be useful:
+    - Is this simply a reply from an email going out from Odoo ? 
+    - Are you using an incoming email server or somehow redirecting? 
+    - Can you provide us with an example of an email that has been correctly forwarded ?
+
+- Providing answers to the following questions: 
+	- Is it a generic issue or is it specific to a use case? If yes, which one exactly? 
+	- Is it working as expected? In case the email is sent using Odoo, the bounce email should reach the Odoo database and display the :ref:`red envelope <red_envelop>`. To be able to investigate, please refer to the subsection envelope is red.
```

Comment:
```suggestion
  - Is it a generic issue or is it specific to a use case? If yes, which one exactly? 
  - Is it working as expected? In case the email is sent using Odoo, the bounce email should reach the Odoo database and display the :ref:`red envelope <red_envelop>`. To be able to investigate, please refer to the subsection envelope is red.
```

Reply:
This was made on purpose :)